platforms/dji/tello/driver.go

```diff
@@ -279,6 +279,17 @@ func (d *Driver) TakeOff() (err error) {
 	return
 }
 
+// Throw & Go support 
+func (d *Driver) ThrowTakeOff() (err error) {
+	buf, _ := d.createPacket(0x5d, 0x48, 0)
```

Comment:
You should probably create a constant for the command byte here, to match the other commands.

Reply:
Oh, sorry, so stupid of me. Can you help me to edit my code?

Reply:
If you make a change to your branch, commit it, and then push to Github, it should update this PR.

Reply:
Also, add the event and handle it...

Reply:
Good point @SMerrony yes, that is also needed.

Reply:
@SMerrony There's no such event, I think. It's just FlightData.ThrowFlyTimer and receives with FlightDataEvent